README.md

```diff
@@ -116,5 +116,6 @@ Static code analysis
 In order for a PR to be accepted the following pylint command needs to result in 10/10 analysis:
 ```
 cd MITuna/tuna
+pylint -f parseable -d duplicate-code --max-args=8 --indent-string '  ' *.py
 pylint -f parseable -d duplicate-code --max-args=8 --indent-string '  ' miopen/*.py example/*.py *.py
```

Comment:
Please take out L119 since it's doing the same thing as L120 is already doing

Reply:
Fixed